ethtxmanager/ethtxmanager.go

```diff
@@ -145,14 +145,42 @@ func pendingL1Txs(URL string, from common.Address, httpHeaders map[string]string
 	return mTxs, nil
 }
 
+// getTxNonce get the nonce for the given account
+func (c *Client) getTxNonce(ctx context.Context, from common.Address) (uint64, error) {
+	// Get created transactions from the database for the given account
+	createdTxs, err := c.storage.GetByStatus(ctx, []MonitoredTxStatus{MonitoredTxStatusCreated})
```

Comment:
Assuming this instance of EthTxNamager is going to work with a single sender, the code below should work.

But as you can see in the node code: https://github.com/0xPolygonHermez/zkevm-node/blob/ace9611c8d4be9a5ee8165a3f5bf303daff1a83b/ethtxmanager/ethtxmanager.go#L65-L68

we had to create a new method to load the monitored txs by status but also by `Sender`, so we can keep track of the nonce for the correct account sending this transaction that is being added.

Since the code is read to handle multiple senders at once, I recommend implementing the GetBySenderAndStatus, that will also work for a single sender, wdyt?

Reply:
This version of the eth-tx-manager is to be used as a library and the sender is retrieved from the keystore provided in the creation of the instance. So the sender is not needed.